### PR TITLE
Update MainWindow.cpp

### PR DESCRIPTION
**Pull request by abhineetagarwal**
Fix memory corruption

### CODE REVIEW

**Comment by abhineetagarwal** [comment]
@joydeep12 : Please review the same

File: platform/wm/rhodes/MainWindow.cpp [hunk 1]
```diff
@@ -1241,8 +1241,11 @@ LRESULT CMainWindow::OnExecuteJSCommand(WORD /*wNotifyCode*/, WORD /*wID*/, HWND
 {
     TNavigateData* nd = (TNavigateData*)hWndCtl;
     if (nd) {
-        LPTSTR wcurl = (LPTSTR)(nd->url);
-        if (wcurl) 
+        wchar_t wcurl[2046];
+	memset(wcurl,'0',2046);
```
Comment (darryncampbell) [inline platform/wm/rhodes/MainWindow.cpp]: What if the javascript command is > 2046 characters?

**Comment by joydeep12** [comment]
Looks good.Ready to be merged.

File: platform/wm/rhodes/MainWindow.cpp (hunk 1 above)
Comment (abhineetagarwal) [inline platform/wm/rhodes/MainWindow.cpp]: Do we have such big command whose character length is more than 2046

Comment (abhineetagarwal) [inline platform/wm/rhodes/MainWindow.cpp]: I dont want to use _tcdup command...

Comment (ats3user) [inline platform/wm/rhodes/MainWindow.cpp]: Do you mean to fill the buffer with the character '0'? Shouldn't it be memset(wcurl,0,2046)? And wchar_t is a wide character type (2 bytes), so the buffer will actually be 2 * 2046 bytes long.

Comment (darryncampbell) [inline platform/wm/rhodes/MainWindow.cpp]: And yes, we do have javascript commands that are > 2046 characters potentially.  One example I know of is the 2.2 Version API but I'm sure it's a limitation will frequently be exceeded.

Comment (abhineetagarwal) [inline platform/wm/rhodes/MainWindow.cpp]: Well in that case... I will be creating the size dynamically.. I will modify the same.

**Comment by abhineetagarwal** [comment]
Fixed... Hence merging the same.

**Comment by abhineetagarwal** [comment]
Thanks @darryncampbell

**Comment by darryncampbell** [comment]
No worries :).  I'm on PTO next week but please feel free to reach out to @geoffday67 (AKA @ats3user ) if there are additional questions on this.

**Comment by abhineetagarwal** [comment]
Ok. Thanks...

**Pull request merged**